src/main/java/chess/controller/RoomController.java

```diff
@@ -0,0 +1,64 @@
+package chess.controller;
+
+import chess.service.CommandService;
+import chess.service.RoomService;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+@Controller
+public class RoomController {
+
+    private final RoomService roomService;
+    private final CommandService commandService;
+
+    public RoomController(RoomService roomService, CommandService commandService) {
+        this.roomService = roomService;
+        this.commandService = commandService;
+    }
+
+    @GetMapping("/")
+    public ModelAndView index() {
+        ModelAndView modelAndView = new ModelAndView("index");
+        modelAndView.addObject("rooms", roomService.findAllRooms());
+        return modelAndView;
+    }
```

Comment:
```suggestion
    @GetMapping
    public String index(Model model) {
        model.addAttribute("rooms", roomService.findAllRooms());
        return "index";
    }
```

이런 방식으로도 가능해요(요즘 주로 선호되는 방식인 것 같네요).

Reply:
view랑 모델이랑 분리해서 보낼 수 있군요! 수정하겠습니다